### DISCUSSION TITLE
Center the text annotation for regions

### DISCUSSION THREAD

**Discussion by jlian2** [katspaugh/wavesurfer.js#3093] [Q&A]
![image](https://github.com/katspaugh/wavesurfer.js/assets/61141271/314463e1-40ae-4679-a487-951f8f4cca48)

As shown in the example, texts are by default on the leftmost. I want them to be centered. How can I achieve this?

**Comment by katspaugh** [comment]
Using [CSS](https://wavesurfer-js.org/examples/#styling.js).
